app/workers/evss/disability_compensation_form/upload_bdd_instructions.rb

```diff
@@ -60,7 +62,7 @@ def file_body
 
       def client
         @client ||= if Flipper.enabled?(:disability_compensation_lighthouse_document_service_provider)
-                      # TODO: create client from lighthouse document service
+                      BenefitsDocuments::Service.new(submission.auth_headers)
```

Comment:
Another similar change to https://github.com/department-of-veterans-affairs/vets-api/pull/13576/files#r1327579513

Noting it looks like all this does is upload a static PDF file that never changes we submit every time